mkdocs.yml

```diff
@@ -131,24 +130,27 @@ nav:
     - Rotate Secrets: "rotate-secrets.md"
     - Terraform: "terraform.md"
     - Authentication: "authentication.md"
-  - Development:
-    - Building: "development/building.md"
-    - Adding a feature: "development/adding_a_feature.md"
-    - Testing: "development/testing.md"
-    - New Kubernetes Version: "development/new_kubernetes_version.md"
-    - Developing using Docker: "development/Docker.md"
-    - Documentation Guidelines: "development/documentation.md"
-    - Hack Directory: "development/hack.md"
-    - How to update kOps API: "development/api_updates.md"
-    - Low level description on how kOps works: "development/how_it_works.md"
-    - Notes on Gossip design: "development/gossip.md"
-    - Notes on master instance sizing: "development/instancesizes.md"
-    - Our release process: "development/release.md"
-    - Releasing with Homebrew: "development/homebrew.md"
-    - Bazel: "development/bazel.md"
-    - Vendoring: "development/vendoring.md"
-    - Ports: "development/ports.md"
-    - Cluster Addons & Manager : "development/addons.md"
+  - Contributing:
+    - Getting Involved and Contributing: "contributing/contributing.md"
+    - New Kubernetes Version: "contributing/new_kubernetes_version.md"
+    - Our Release Process: "contributing/release.md"
+    - Releasing With Homebrew: "contributing/homebrew.md"
+    - Updating The Default Base AMI: "contributing/update_ami_versions.md"
```

Comment:
I do like that we have a Contributing section and a Development subsection and that it makes sense that Development is a subset of Contributing. I'm not sure it makes sense to have the entire Development section along side this handful of pages though, maybe we can put them under their own Subsection? I'm trying to think of a good name for it though... maybe some combination of these words:

common, repeat, periodic, release, processes, instructions, steps


but that would leave us with:
```
Contributing
- Some New Section
- Development
```

which might be awkward on the left side of the webpage to only have two menu items within Contributing. I don't have any good ideas here :(

Reply:
Maybe we should just rename Development to Contributing :).

Reply:
> Maybe we should just rename Development to Contributing :).

Yeah I like that idea. I'll give that a try and see what it looks like

Reply:
Pushed up! 😄 
https://deploy-preview-10455--kubernetes-kops.netlify.app/